content/blog/2022/12/community-meeting.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Community meeting is coming
+authors:
+- Olivier Vernin
+date: 2022-12-21
+---
+
+Community meetings have been a recurring demand from different sides and with the new year approaching,
+It's time to take our first good resolution.
+
+To improve community feedback, The Kubewarden project decided to organize a monthly community meeting.
+The first community meeting to be held is scheduled for January 12th, 2023 at 4 PM UTC.
+
+On top of the GitHub Discussion, GitHub issues, and the Kubewarden channel on the CNCF Slack channel, the community meeting is an additional opportunity to meet and discuss Kubewarden and shape its future together.
+
+Feel free to join us at https://zoom.us/j/92928111886
```

Comment:
Ideally, I would not link this here since it could be misused. Y'all should create a google group with the Kubewarden DL & create a recurring monthly invite that people can receive in their inbox.

Reply:
I hesitated to mention it here for the exact reason you just explain

Reply:
I am removing the link

Reply:
```suggestion
Feel free to join us
```